PhysicsTools/NanoAOD/python/taus_cff.py

```diff
@@ -167,6 +169,13 @@ def _tauId8WPMask(pattern,doc):
                  idAntiEle = _tauId5WPMask("againstElectron%sMVA6", doc= "Anti-electron MVA discriminator V6 (2015)")
     )
```

Comment:
These lines replace the PSet _mvaAntiEVars in all cases it is used. Therefore I suggest to remove _mvaAntiEVars above and only keep this modifier.